pingcap/br
# err_msg: clarify some ambiguous error message

### What problem does this PR solve? 
Keeping improve BR errors and logs [#1055 ](https://github.com/pingcap/br/issues/1055)

Some error message in BR is ambiguous and misleading. 
i.e. 
- If tikv has no permission to read from & write to backup storage, error message cannot clarify whether error occur on br side or tikv side. Users may think why I got "permission denied"? I'm running BR as root!
- If user want to restore tables or databases that have not been backup, error should occur.

Clarify the summary info including:
- `data-size => total kv size`
- `size => backup data size(after compressed)`



### What is changed and how it works?
1. Adjust the backup and restore summary:
- the `data-size` field has been changed to `total-kv-size`
- the `size` field in backup summary has been changed to `backup data size(after compressed)` 
- the `size` field in restore summary has been changed to `restore data size(after decompressed)`

2. Some error messages are changed, and will clarify what is going on when error occur.

- Original error message is as following
```
[2021/05/18 16:37:28.517 +08:00] [ERROR] [push.go:147] ["backup occur unknown error"] [error="Io(Os { code: 13, kind: PermissionDenied...})"] [stack="github.com/pingcap/br/pkg/backup.(*pushDown).pushBackup\n\t/home/zwj/log_improve/br/pkg/backup/push.go:147\ngithub.com/pingcap/br/pkg/backup.(*Client).BackupRange\n\t/home/zwj/log_improve/br/pkg/backup/client.go:540\ngithub.com/pingcap/br/pkg/backup.(*Client).BackupRanges.func2.1\n\t/home/zwj/log_improve/br/pkg/backup/client.go:476\ngithub.com/pingcap/br/pkg/utils.(*WorkerPool).ApplyOnErrorGroup.func1\n\t/home/zwj/log_improve/br/pkg/utils/worker.go:63\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/root/go/pkg/mod/golang.org/x/sync@v0.0.0-20201020160332-67f06af15bc9/errgroup/errgroup.go:57"]
```

- For now, the error message output is as following: 
```
[2021/05/19 15:43:43.305 +08:00] [ERROR] [push.go:175] [error="[BR:KV:ErrKVStorage]tikv storage occur I/O error: I/O permission denied error occurs on TiKV Node(store id: 1; Address: 127.0.0.1:20162)"] ["work around"="please ensure tikv has permission to read from & write to the storage."] [stack="github.com/pingcap/br/pkg/backup.(*pushDown).pushBackup\n\t/home/zwj/log_improve/br/pkg/backup/push.go:175\ngithub.com/pingcap/br/pkg/backup.(*Client).BackupRange\n\t/home/zwj/log_improve/br/pkg/backup/client.go:540\ngithub.com/pingcap/br/pkg/backup.(*Client).BackupRanges.func2.1\n\t/home/zwj/log_improve/br/pkg/backup/client.go:476\ngithub.com/pingcap/br/pkg/utils.(*WorkerPool).ApplyOnErrorGroup.func1\n\t/home/zwj/log_improve/br/pkg/utils/worker.go:63\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/root/go/pkg/mod/golang.org/x/sync@v0.0.0-20201020160332-67f06af15bc9/errgroup/errgroup.go:57"]
```

3. When user want to restore tables or databases that have not been backup, error should occur.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. If user want to restore a table or database that has not been backup, an error should occur to notice the user
![image](https://user-images.githubusercontent.com/23160968/118946455-fc07ea00-b988-11eb-95c7-cdfed0f781d8.png)
![image](https://user-images.githubusercontent.com/23160968/118946524-0aee9c80-b989-11eb-9f06-ce6215fb2db8.png)
2. If user restore a empty table, the summary should notice the user there is nothing to restore.
![image](https://user-images.githubusercontent.com/23160968/118470386-38450b80-b739-11eb-8e9d-42aafc5fc1cd.png)
3. Summary message have improve, data-size => total-kv-size and size => backup data size(after compressed)
![image](https://user-images.githubusercontent.com/23160968/118622459-ec0dd000-b7f9-11eb-952d-e90c47879624.png)


 - Integration test

Code changes

 - Has exported variable/fields change

### Release Note

 - No realease note
